plugins/spender/multifundchannel.c

```diff
@@ -1340,9 +1366,13 @@ after_fundpsbt(struct command *cmd,
 	mfc->psbt = psbt_from_b64(mfc,
 				  buf + field->start,
 				  field->end - field->start);
+
```

Comment:
Gratuitous line add.  Generally a bad habit, but here it's borderline...

Reply:
removed